cli/pkg/cmd/create/create.go

```diff
@@ -78,69 +88,69 @@ func create(cmd *cobra.Command, args []string) {
 	fmt.Fprintln(w)
 	fmt.Fprintln(w, providerName, "provider output:")
 	fmt.Fprintln(w)
-	context, err := provider.Provision(providerConfig, dir)
+	context, err := provider.Provision(providerConfig, workspace.ConfigDir{Path: installDir})
 	fmt.Fprintln(w)
 	if err != nil {
-		console.Exit("could not provision cluster: ", err)
-	}
-
-	fmt.Println("✅ Cluster provisioned")
-
-	clientset, err := context.GetClientSet()
-	if err != nil {
-		console.Exit("could not get clientset: ", err)
+		console.Exit("could not install Airy: ", err)
 	}
+	fmt.Println("✅ Airy Installed")
+	if providerName != "aws" { // TEMP fix to keep minikube working, but will soon change
```

Comment:
This we must have until we also change Minikube. But maybe put `if providerName == "minikube"` better, as it depends on that provider in particular and for "all the rest" should be otherwise.